### PR TITLE
infra: improve read scalability

### PR DESCRIPTION
**Pull request by renebarbosafl**
This PR enable chain-sync and marlowe-sync apps to read from a PostgreSQL replica

### CODE REVIEW

**Comment by shlevy** [comment]
We should also be able to scale up everything except the indexers.

**Comment by renebarbosafl** [comment]
@shlevy Ok!
Updated in [22e864d](https://github.com/input-output-hk/marlowe-cardano/pull/740/commits/22e864d087f3601f26dc15e16e52dc98e51e65ed)

File: deploy/marlowe-runtime/templates/marlowe-web-server.yaml [hunk 1]
```diff
@@ -34,6 +34,11 @@ spec:
           port: 3780
           protocol: TCP
       traits:
+      - type: cpuscaler
+        properties:
+          min: 1
+          max: 10
+          cpuUtil: 50
```
Comment (jhbertra) [inline deploy/marlowe-runtime/templates/marlowe-web-server.yaml:41]: We can't scale the webserver until we address the in-memory state issue

Comment (renebarbosafl) [reply deploy/marlowe-runtime/templates/marlowe-web-server.yaml]: Disabled on [3b3cb48](https://github.com/input-output-hk/marlowe-cardano/pull/740/commits/3b3cb483b96c2b531875195719ebaec8c2fa5e02)

**Comment by renebarbosafl** [comment]
@jhbertra As we discussed, [this](https://input-output.atlassian.net/browse/PLT-8483) is the bug I've seen when tried to enable chain-sync to read from read-only postgresql replicas.

**Comment by renebarbosafl** [comment]
I am closing this PR as it's stalled.
When https://input-output.atlassian.net/browse/PLT-8483 is completed, we create another PR.

**Pull request closed without merging**